### ISSUE TITLE
Tesla app

### ISSUE THREAD

**Issue by terrorist96** [AdguardTeam/AdguardFilters#168037]
### Prerequisites

- [X] This site DOES NOT contain sexually explicit material, otherwise use [NSFW-specific form](./new?template=bug_report_NSFW.yml);
- [X] Filters were updated before reproducing an issue;
- [X] AdGuard product version is up-to-date;
- [X] Browser version is up-to-date;
- [X] If the site or app is broken, disabling AdGuard protection resolves an issue.

### What product do you use?

AdGuard for Android

### AdGuard version

4.2

### What type of problem have you encountered?

Website or app doesn't work properly

### Which browser(s) do you use?

Chrome, the problem is with app, not a site

### Which device do you use?

Mobile

### Where is the problem encountered?

tesla app

### Ad Blocking

AdGuard Base filter, AdGuard Mobile Ads filter

### Social Widgets

AdGuard Social Media filter

### Annoyances

AdGuard Cookie Notices filter, AdGuard Popups filter, AdGuard Mobile App Banners filter, AdGuard Other Annoyances filter, AdGuard Widgets filter

### Add your comment and screenshots

Please see AdGuard Support Request #839023. The Tesla app needs to be allowlisted or the issue fixed. 

### Privacy

- [X] I agree to follow this condition

**Comment by terrorist96** [comment]
Tesla is not popular enough? 😂

**Comment by Alex-302** [comment]
@terrorist96 What exactly breaks the app - filters, dns filtering or filtering, when all filters and dns filtering is disabled?

**Comment by terrorist96** [comment]
> @terrorist96 What exactly breaks the app - filters, dns filtering or filtering, when all filters and dns filtering is disabled?

I don't know. I wasn't able to see anything in the list of blocked items being the culprit. All I know is that disabling "route traffic through Adguard" for the Tesla app resolved the issue. More details are in the above referenced ticket, which the person (or seemingly AI) that responded was not very helpful with. Are you able to see the contents of that ticket?

**Comment by Alex-302** [comment]
Please reply in support emails, or write to https://github.com/AdguardTeam/AdguardForAndroid

**Comment by Alex-302** [comment]
> or seemingly AI

Not AI.

**Comment by terrorist96** [comment]
> Please reply in support emails, or write to https://github.com/AdguardTeam/AdguardForAndroid

Ok opened https://github.com/AdguardTeam/AdguardForAndroid/issues/5108

> > or seemingly AI
> 
> Not AI.

Coulda fooled me. The responses were generic and not specific to the issue. Tried asking what the color of the sky is during the day two times to test if I'm talking to a human or bot and didn't get the answer either time. Closed my ticket without resolving the issue so that's why I tried GitHub.
